### PR TITLE
Nueva era

### LINKED ISSUE #440: There are some pages that render wrong
Click, for instance, in any reference to the name Level.

### PR DESCRIPTION
**Pull request by jonaprieto**
This PR improves the website in many ways. The resulting website'd be like this one: https://jonaprieto.github.io/agda-unimath/

- Closes #440 
- Adds some progress #195 

Some new features added by this PR:

- Table of contents (navigation sidebar),
- list of contributors on Github,
- search bar,
- light and dark themes, and
- print option.

We got all this by using Mdbook. I hope this helps to prove more theorems and people get interested in formalising more stuff using agda-unimath.

Of course, this is a starting point, and I expect many fixes when I find time:
Ideas that I'm sure we can get:

- Better support for latex using Katex.
- Bibtex support.
- Tikz support. 
- Copyable button for Agda snippets.
- Share options.
- Better print output.
- Hide imports at will.

### CODE REVIEW

**Comment by fredrik-bakke** [comment]
An observation:
While browsing the new website I noticed that the page for `foundation-core.coproduct-types` was missing. When looking, I found that it was not imported in `src/README.lagda.md`, however, with the current website the page is at least accessible through other links, like when coming from `foundation.coproduct-types`.

**Comment by fredrik-bakke** [comment]
Also, the title of the index says "Statement of inclusion". Should this be changed to "Univalent mathematics in Agda"?

File: src/SUMMARY.lagda.md [hunk 1]
```diff
@@ -0,0 +1,1021 @@
+# SUMMARY
+
+- [README](README.md)
+- [Statement of Inclusion](STATEMENT-OF-INCLUSION.md)
+- [Everything](everything.md)
+
+---
+
+# Category theory
```
Comment (EgbertRijke) [inline src/SUMMARY.lagda.md:9]: Shouldn't headings that aren't the title of a page be with `##`?

**Comment by EgbertRijke** [comment]
It also looks like the titles of individual pages aren't displayed at the top. For example, the page

https://jonaprieto.github.io/agda-unimath/foundation.automorphisms.html

should display the tile "Automorphisms" at the top, but it doesn't.

File: src/SUMMARY.lagda.md (hunk 1 above)
Comment (jonaprieto) [reply src/SUMMARY.lagda.md]: This file, SUMMARY, works a bit different. In general, we should use `##` for subtitles yes, here are used to make sections/parts. I changed it anyway, and restructured a bit more.

**Comment by jonaprieto** [comment]
Oh Good catch, easy to fix.

**Comment by EgbertRijke** [comment]
Also, clicking "Agda-UniMath" at the top currently does nothing. It would be nice if it would bring us back to the home page.

**Comment by fredrik-bakke** [comment]
I hope you don't mind me coming with yet another opinion, but I think it would be nice to use the font that GitHub displays Agda code in as it is more uniform in style than the one that is currently used on the new website. Compare for instance the lambdas and Sigmas in the current font with the ones displayed on GitHub. Would this be easy to change?

**Comment by fredrik-bakke** [comment]
Oh, another thing I just noticed. Excessively long lines of code escape the boxes. See e.g. in
https://jonaprieto.github.io/agda-unimath/finite-group-theory.sign-homomorphism.html

**Comment by EgbertRijke** [comment]
I really like Jonathan's new way of displaying the code. For me, it doesn't have to be identical to github's formatting. It looks pretty this way.

**Comment by EgbertRijke** [comment]
On mobile, basically every line escapes the boxesimage

**Comment by fredrik-bakke** [comment]
For sure, I like the formatting, and the font doesn't necessarily have to be the same as GitHub's. But there's something about these symbols in particular that look jarring to me. Also, square brackets look like they are aligned too low.

I don't have any other suggestion than that I like GitHub's font better than the currently used. I also like the fonts that are used in VSCode. This seems to be `Menlo, Monaco, 'Courier New', monospace` in my current configuration.

**Comment by EgbertRijke** [comment]
Ah, I agree with you. That does look off.

**Comment by EgbertRijke** [comment]
On my computer the square brackets display correctly:

Screenshot 2023-02-13 at 18 43 02

**Comment by jonaprieto** [comment]
> Ah, I agree with you. That does look off.

I fixed the font a few minutes ago. ;)

**Comment by jonaprieto** [comment]
> On mobile, basically every line escapes the boxesimage

This should be fixed by the latest commit, can you confirm it?

**Comment by EgbertRijke** [comment]
Yes, it looks amazing, even on the phone!

**Comment by fredrik-bakke** [comment]
There seems to be left a margin for a vertical scrollbar in each of the Agda code windows. If I'm not mistaken, this is never needed and can be hidden, no?

**Comment by jonaprieto** [comment]
> Also, clicking "Agda-UniMath" at the top currently does nothing. It would be nice if it would bring us back to the home page.

I couldn't find an easy solution to this. We probably need to define our custom theme. Let's do this later.

**Comment by jonaprieto** [comment]
> There seems to be left a margin for a vertical scrollbar in each of the Agda code windows. If I'm not mistaken, this is never needed and can be hidden, no?
> 
> Screenshot 2023-02-13 at 19 06 40

I fixed it, can you confirm it @fredrik-bakke?

**Comment by fredrik-bakke** [comment]
> > There seems to be left a margin for a vertical scrollbar in each of the Agda code windows. If I'm not mistaken, this is never needed and can be hidden, no?
> > Screenshot 2023-02-13 at 19 06 40
> 
> I fixed it, can you confirm it @fredrik-bakke?

Yes, looks great! 😄

File: src/foundation-core/truncation-levels.lagda.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
----
-title: Truncation levels
+{{#title  Truncation levels
```
Comment (EgbertRijke) [inline src/foundation-core/truncation-levels.lagda.md:1]: Braces are wrong

File: src/foundation/cantor-schroder-bernstein-escardo.lagda.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
----
-title: The Cantor-Schröder-Bernstein-Escardó theorem
+{{#title  The Cantor-Schröder-Bernstein-Escardó theorem
```
Comment (EgbertRijke) [inline src/foundation/cantor-schroder-bernstein-escardo.lagda.md:1]: Braces are wrong

File: src/foundation/dependent-paths.lagda.md [hunk 4]
```diff
@@ -1,5 +1,4 @@
----
-title: Dependent paths
+{{#title  Dependent paths
```
Comment (EgbertRijke) [inline src/foundation/dependent-paths.lagda.md:1]: Braces are wrong

File: src/foundation/equational-reasoning.lagda.md [hunk 5]
```diff
@@ -1,5 +1,4 @@
----
-title: Equational reasoning
+{{#title  Equational reasoning
```
Comment (EgbertRijke) [inline src/foundation/equational-reasoning.lagda.md:1]: Braces are wrong

File: src/foundation/hilberts-epsilon-operators.lagda.md [hunk 6]
```diff
@@ -1,6 +1,5 @@
 ---
-title : Hilbert's ε-operators
----
+title : Hilbert's ε-operators}}
```
Comment (EgbertRijke) [inline src/foundation/hilberts-epsilon-operators.lagda.md:2]: Title is wrong

File: src/foundation/shifting-sequences.lagda.md [hunk 7]
```diff
@@ -1,6 +1,5 @@
 ---
-title : Shifting sequences
----
+title : Shifting sequences}}
```
Comment (EgbertRijke) [inline src/foundation/shifting-sequences.lagda.md:2]: Title is wrong

File: src/foundation/truncation-levels.lagda.md [hunk 8]
```diff
@@ -1,5 +1,4 @@
----
-title: Truncation levels
+{{#title  Truncation levels
```
Comment (EgbertRijke) [inline src/foundation/truncation-levels.lagda.md:1]: Braces are wrong

Comment (jonaprieto) [reply src/foundation/truncation-levels.lagda.md]: done

File: src/foundation/equational-reasoning.lagda.md (hunk 5 above)
Comment (jonaprieto) [reply src/foundation/equational-reasoning.lagda.md]: done

File: src/foundation/cantor-schroder-bernstein-escardo.lagda.md (hunk 3 above)
Comment (jonaprieto) [reply src/foundation/cantor-schroder-bernstein-escardo.lagda.md]: Done

File: src/ring-theory/dependent-products-rings.lagda.md [hunk 9]
```diff
@@ -1,6 +1,4 @@
----
-title:
----
+#
```
Comment (EgbertRijke) [inline src/ring-theory/dependent-products-rings.lagda.md:1]: This title is empty

File: src/ring-theory/maximal-ideals-rings.lagda.md [hunk 10]
```diff
@@ -1,6 +1,4 @@
----
-title:
----
+#
```
Comment (EgbertRijke) [inline src/ring-theory/maximal-ideals-rings.lagda.md:1]: Title is empty

File: src/set-theory/uncountable-sets.lagda.md [hunk 11]
```diff
@@ -1,5 +1,4 @@
----
-title: Uncountable sets
+{{#title  Uncountable sets
```
Comment (EgbertRijke) [inline src/set-theory/uncountable-sets.lagda.md]: Braces/title are wrong

File: src/synthetic-homotopy-theory/multiplication-circle.lagda.md [hunk 12]
```diff
@@ -1,5 +1,4 @@
----
-title: The multiplication operation on the circle
+{{#title  The multiplication operation on the circle
```
Comment (EgbertRijke) [inline src/synthetic-homotopy-theory/multiplication-circle.lagda.md]: Braces/title wrong

**Pull request merged**